docs/diagnostics/PX1007.md

```diff
@@ -15,13 +15,28 @@ The code fix does one of the following:
  - Adds the `summary` XML tags, in which you type the description of the code item
  - Adds the `exclude` XML tag, which excludes the item from the API and DAC References
 
-The new diagnostic is available only if the value of **Tools > Options > Acuminator > Code Analysis > Enable PX1007 diagnostic** is set to _True_. By default, the value of this option is _False_. 
-
-This diagnostic is intended for Acumatica ERP developers, however it can be used by any developers to provide documentation of source code.
-
-The diagnostic is currently displayed for all public entities (classes, interfaces, structs, enumerations, and delegates) and DAC property fields no matter in which namespace they are located. The diagnostic is not displayed for other fields, properties, and methods of public entities. 
-
-The diagnostic is not displayed for child classes if their parent class has the `exclude` XML tag.
+The new diagnostic is available only if the value of **Tools > Options > Acuminator > Code Analysis > Enable PX1007 diagnostic** is _True_. By default, the value of this option is _False_. 
+
+This diagnostic is intended for Acumatica ERP developers; however, it can be used by any developer to provide documentation of the source code.
+
+The diagnostic is currently displayed for the following public items:
+ - Classes that are not DACs 
+ - DACs with the `PXCacheName` attribute
+ - DAC property fields
+ - Interfaces
+ - Structs
+ - Enumerations
+ - Delegates
+
+The diagnostic is not displayed for the following items: 
+ - Fields, properties, and methods of public entities (except DAC property fields).
```

Comment:
I would say type members of public entities unless it is a nested public type (class/struct/enum/interface/delegate type) or a public DAC property
Because there are other kinds of type members - instance and static constructors, events

Reply:
modified